store/src/java-test/com/zimbra/cs/service/mail/GetMsgTest.java

```diff
@@ -54,11 +53,16 @@
 import com.zimbra.cs.mailbox.MailboxTestUtil;
 import com.zimbra.cs.mailbox.calendar.Invite;
 import com.zimbra.cs.mailbox.calendar.ZOrganizer;
-
-
+import com.zimbra.soap.JaxbUtil;
+import com.zimbra.soap.mail.message.CreateMountpointRequest;
+import com.zimbra.soap.mail.message.FolderActionRequest;
+import com.zimbra.soap.mail.message.GetMsgRequest;
+import com.zimbra.soap.mail.type.FolderActionSelector;
+import com.zimbra.soap.mail.type.MsgSpec;
+import com.zimbra.soap.mail.type.NewMountpointSpec;
 
 public class GetMsgTest {
-
+    private static final String desc = "The following is a new meeting " + "request";
```

Comment:
Any reason this isn't just a single String?  Unsure why it needs a + in there.  I see the original line had one.